docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -60,11 +60,17 @@ alluxio.master.mount.table.root.ufs=<STORAGE_URI>
   mount to the Alluxio root.
   This shared storage system must be accessible by the master node and all worker nodes.
   Examples include `alluxio.master.mount.table.root.ufs=hdfs://1.2.3.4:9000/alluxio/root/`, or 
-  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`.
+  `alluxio.master.mount.table.root.ufs=s3://bucket/dir/`
+  (See [this tutorial](https://docs.alluxio.io/os/user/stable/en/ufs/S3.html#running-alluxio-locally-with-s3) 
+  for more details about configuring Amazon S3 as Alluxio's under storage system).
+
+Then, append the host name of all the Alluxio master nodes into `conf/masters` 
```

Comment:
both of these changes are explained in https://docs.alluxio.io/os/user/stable/en/contributor/Documentation-Conventions.html
- remove `Then, ` -> imperative mood
- change `host name` to `hostname` -> terminology table
  - actually it should be `hostnames` because it is followed by `all the Alluxio master nodes`

also, i think the current description is not precise enough for someone new to understand. one might append the hostnames separated by a space in the file. provide an example with some dummy hostnames as placeholders